### PR TITLE
Improve accessibility of filters on desktop

### PR DESCRIPTION
**Pull request by LucasCioffi**
Page jumps vertically on filter change: WWW-530
Ensure the filtering experience is accessible WWW-465
Implement design and functionality of tabs for desktop WWW-517

### CODE REVIEW

**Comment by LucasCioffi** [comment]
@ktp242 this is the work that Daniel and I did together; we can review together if that's helpful.

File: src/app/components/Results/Results.jsx [hunk 1]
```diff
@@ -233,7 +232,7 @@ class Results extends React.Component {
   }
 
 
-  selectedTab(tabIdValue){
+  saveSelectedTabValue(tabIdValue){
```
Comment (LucasCioffi) [inline src/app/components/Results/Results.jsx]: We thought this was a more accurate name so we changed it.

**Comment by ktp242** [comment]
If I can press down art the end of the filters and get into the result section, should I also be able to press up and back to the filters?

**Comment by LucasCioffi** [comment]
> If I can press down art the end of the filters and get into the result section, should I also be able to press up and back to the filters?

@ktp242 that's a good thought.  It wasn't in the requirements, so I'll check with @wlla.

**Comment by LucasCioffi** [comment]
@ktp242 I'm going to leave the remaining ESLint errors since they relate to the section of code @nnat425 is editing for the mobile version of the filter.
screen shot 2018-12-04 at 11 57 59 am

Does this cover all the issues you brought up?

File: package.json [hunk 2]
```diff
@@ -8,7 +8,8 @@
     "dist": "NODE_ENV=production webpack --config webpack.config.js",
     "test": "node_modules/.bin/mocha --compilers js:babel-core/register",
     "accessibility-check": "loadA11y=true node index",
-    "eb-start": "npm run dist && npm run start"
+    "eb-start": "npm run dist && npm run start",
+    "lint": "eslint ./"
```
[the remark this answers is not in the document]
Comment (LucasCioffi) [reply package.json]: Thanks, @ktp242, I added a new ticket for better linting: https://jira.nypl.org/browse/WWW-553

**Pull request merged**